src/main/java/com/hyperlocal/server/Data/Shop.java

```diff
@@ -1,69 +1,71 @@
 package com.hyperlocal.server.Data;
 
-import com.hyperlocal.server.Utilities;
+import java.io.Serializable;
+import java.math.BigDecimal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.github.jasync.sql.db.RowData;
+import com.google.auto.value.AutoValue;
 import com.google.gson.JsonObject;
-import java.math.BigDecimal;
 
-public class Shop {
-    public Long shopID;
-    public String merchantID;
-    public String shopName, addressLine1, typeOfService;
-    public Double latitude, longitude;
+@AutoValue
+public abstract class Shop implements Serializable{
+
+    @JsonProperty public abstract Long shopID();
+    @JsonProperty public abstract String merchantID();
+    @JsonProperty public abstract String shopName();
+    @JsonProperty public abstract String addressLine1();
+    @JsonProperty public abstract String typeOfService();
+    @JsonProperty public abstract Double latitude();
+    @JsonProperty public abstract Double longitude();
     
-    public Shop() {
-      this.shopID = 1L;
-      this.merchantID = "1";
-      this.shopName = "Test";
-      this.addressLine1 = "TEST";
-      this.typeOfService = "TEST";
-      this.latitude = 32.32;
-      this.longitude = 32.32;
+    public static Shop create() {
+      return new AutoValue_Shop(
+        0L,
+        "1",
+        "",
+        "",
+        "",
+        0.00,
+        0.00
```

Comment:
you should specify values like below, so that value of the number can be infereed.

/* shopId = */ 0